project/BuildInfo.scala

```diff
@@ -0,0 +1,30 @@
+import com.typesafe.sbt.GitPlugin.autoImport.git
+import sbt.Keys._
+import sbt.Project
+import sbtbuildinfo.BuildInfoKeys._
+import sbtbuildinfo.{BuildInfoKey, BuildInfoPlugin}
+
+object BuildInfo {
+
+  implicit class BuildInfoOps(project: Project) {
+
+    //TODO Probably API and Lorre should have BuildInfo structure split across both of these modules
+    // Currently name is HardCoded, should be changed, once used by multiple modules
```

Comment:
Can't we simply read the name of the `Project` object?
In any case I agree with your analysis, yet not sure about splitting into separate module-specific objects.

Reply:
Not really, because name from the project would be different than "Conseil". It will be "Conseil-api" and "Conseil-lorre" which in my opinion is exactly how it should work, but since I wanted to reduce number of changes I decided to hardcode that. We **may** introduce different version for API that Lorre part, but here, it will be under the same name, so it might confuse community, since asking API for version will give different value than Lorre for the same project name.

Reply:
Well, I see that we need to figure out how to handle the versioning if modules can be published separately and evolve as such...
It's a whole new level of complexity that I wouldn't add now.

Reply:
All right, just wanted to point out my reasoning. I also think we should go with current behavior for the time being.